sources/packages/web/src/views/institution/AppInstitution.vue

```diff
@@ -88,10 +114,34 @@ export default {
       AppConfigService.shared.logout(ClientIdType.INSTITUTION);
     };
 
+    const togleUserMenu = (event: any) => {
+      userOptionsMenuRef.value.toggle(event);
+    };
+
+    userMenuItems.value = [
+      {
+        label: "Notifications Settings",
+        icon: "pi pi-bell",
+        command: () => {
+          AppConfigService.shared.logout(ClientIdType.INSTITUTION);
```

Comment:
The notifications settings should not point to the logout. I would recommend adding a placeholder page in the same way that we did for Students.

Reply:
Agreed. Forgot to remove that action for notifications icon.